### ISSUE TITLE
Tab oder not saved (after restart)

### ISSUE THREAD

**Issue by tentaurus** [amanharwara/altus#139]
**Describe the bug**
Tab oder is not sved after restart

**To Reproduce**

1.

**Expected behavior**
Tab order should be saved

**Screenshots**
If applicable, add screenshots to help explain your problem:

**Information:**
4.1.0

**Additional context**

**Comment by amanharwara** [comment]
Working correctly on my end.

**Comment by tentaurus** [comment]
I checked again with 4.2.0 - same result!
Also: it always starts in the middle of my (first) screen, not at the last position (pinned at the right side of my second screen).
Both works fine with 3.24.
